mkosi/resources/man/mkosi.1.md

```diff
@@ -2858,6 +2799,84 @@ script. When all three are enabled together turn-around times for
 complete image builds are minimal, as only changed source files need to
 be recompiled.
 
+# TOOLS TREES
+
+Tools trees are a secondary image that mkosi can use to build the actual images. This is useful to make image
+builds more reproducible, but also allows to use newer tooling, that is not yet available in the host
+distribution running mkosi.
+
+Tools trees can be provided via the `ToolsTree=` option, the `mkosi.tools` directory or built automatically
+by mkosi if set to `ToolsTree=default`. For most use cases setting it is sufficient to use the default tools
+trees and the use of a tools tree is recommended.
+
+Tools trees can be further customized via the different `ToolsTree*=` variables as well as the
+`mkosi.tools.conf` configuration file or directory. The output format for tools trees cannot currently be
+changed via configuration files. Please use the `-t` option on the commandline.
```

Comment:
This still comes out of nowhere. We have to mention first that it's possible at all to run something like `mkosi --include=mkosi-tools`